aws/mlcluster-vpc.template

```diff
@@ -93,6 +99,16 @@ Parameters:
       - standard
       - gp2
     Default: gp2
+  VolumeEncryption:
+    Description: Whether enable volume encryption
+    Type: String
+    AllowedValues:
+      - enable
+      - disable
+    Default: disable
+  VolumeEncryptionKey:
+    Description: The key ID of AWS KMS key to encrypt volumes
+    Type: String
```

Comment:
If user doesn't provide a key ID, AWS will use default master key automatically. In my test, it will use aws/ebs key.

Reply:
What I meant here is, this is a parameter that doesn't have a default value. That means users have to provide a value. This is not necessary. If this is an optional field for user, then we provide them a default value.

Reply:
I see what you meant. However, since the type of this parameter is *string* and when it is shown on the web page, it will have a default value which is an empty string. User can leave this parameter as an empty string. Actually, I use the empty string to identify *default*. I will add an empty string default value to make it clearer.